mtgjson5/set_builder.py

```diff
@@ -1396,8 +1414,15 @@ def add_card_kingdom_details(mtgjson_set: MtgjsonSetObject) -> None:
     :param mtgjson_set: MTGJSON Set
     """
     LOGGER.info(f"Adding CK details for {mtgjson_set.code}")
-    translation_table = CardKingdomProvider().get_scryfall_translation_table()
-
+    if global_cache.card_kingdom_map is None:
+        LOGGER.warning("Card Kingdom cache not loaded")
```

Comment:
Might want to differentiate the warnings as it's not clear if the cache was not set or the ttable was not set